server/src/sim/utils.py

```diff
@@ -223,13 +225,25 @@ def dumpfile(variable):
         inds = argsort(printnames)
     else:
         inds = argsort(printbytes)
-    
+
     for v in inds:
         print('Variable %s is %s' % (printnames[v], printsizes[v]))
-    
+
```

Comment:
checkmem is just a helper function, not even used on the BE, but useful for e.g. looking at where the JSON files blow out in size